src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/form_div_layout.html.twig

```diff
@@ -320,7 +320,7 @@
 
 {%- block form_end -%}
   {%- if not render_rest is defined or render_rest -%}
-    {{ form_rest(form) }}
+    {{ form_rest(form, { render_hook: false }) }}
```

Comment:
this prevents `form_rest()` from rendering the hook twice.

Reply:
I'm wondering maybe we should do the opposite to avoid side effects?
Also if we do it this way, does this mean if you forget to add `form_rest` in the form the hook won't be rendered?

Reply:
> I'm wondering maybe we should do the opposite to avoid side effects?

what do you mean by that?

> if you forget to add form_rest in the form the hook won't be rendered?

yes

Reply:
> > I'm wondering maybe we should do the opposite to avoid side effects?
> 
> what do you mean by that?

I meant instead of disabling the feature with the parameter we could use `form_rest(form, { render_hook: true })` instead and explicitly enable it so that if we use a simple `form_rest(form)` the hook is not rendered

> 
> > if you forget to add form_rest in the form the hook won't be rendered?
> 
> yes

ok, so we will have to modify every existing form to add `form_rest` if we want to add the hook?

Reply:
>  meant instead of disabling the feature with the parameter we could use form_rest(form, { render_hook: true }) instead and explicitly enable it so that if we use a simple form_rest(form) the hook is not rendered

The idea was to make it render the hook automatically whenever `form_rest` is used, but maybe it's smart not render it automatically and control it with a parameter :thinking: IMO one more opinion would be great, ping @matks @sarjon 

> ok, so we will have to modify every existing form to add form_rest if we want to add the hook?

yes. Also, every form should have a `form_rest` already rendered, unless developers missed it (usually by mistake)

Reply:
Well here seems like `form_rest` is called automatically by `form_end`

So if I understand the current implementation, if you didn't forget to add `form_rest` in your form, it will be called one time with the hook (by default) and a second time by form_end (without the hook)

And of course you still have the possibility to prevent the hook in your form with an explicit `{{ form_rest(form, { render_hook: false }) }}` This is a bit clearer for me, sorry I wasn't sure about the behavior

It seems to me that this approach is the more user friendly and the less intrusive at the same time, it's a good compromise. But as you said let's confirm it with the others and maybe @eternoendless when he comes bak from holiday (end of the week)